template/extras/src/ts/mappers/post.mapper/with-prisma.ts

```
@@ -0,0 +1,12 @@
+import { Prisma } from '@prisma/client';
+
+const postMapper = (post: Prisma.PostGetPayload<{}>) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)